src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -51,7 +52,12 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         commandResult = command.execute(model);
 
         try {
-            storage.saveAddressBook(model.getAddressBook());
+            if (command.getCommandType().equals("personCommand")) {
```

Comment:
Instead of this, you can consider using the built in instanceof method to simplify the code. Not that major, but good to have